pkg/reconciler/subscription/subscription_test.go

```diff
@@ -129,6 +129,8 @@ var (
 		Namespace:  testNS,
 		Name:       channelName,
 	}
+
+	dlsURI = apis.HTTP(dlcDNS)
```

Comment:
If I am understanding this right dlsURI is intended to be used as a value for dead letter sinks at subscriptions, while dlc is the one for channels.

By having dlsURI and dlcURI pointing to the same URL we might be testing it wrong when having scenarios where DLS for both channels and subscriptions are present. We won't know which one is being used since both are the same.

Could we have something like?

```go
    dlsDNS = "dls.mynamespace.svc." + network.GetClusterDomainName()
    dlsURI = apis.HTTP(dlsDNS)
```

Reply:
Ok let me try to fix that on the whole file, cause right now we just have de dlcUri and Dns and is being used for channels and subscriptions altogether